src/include/miopen/conv/heur/heur.hpp

```diff
@@ -141,7 +141,7 @@ struct ConvHeur
 
         std::sort(sort_res.begin(), sort_res.end(), cmp);
```

Comment:
Sorting is not required anymore (otherwise what is the benefit of returning AI Heuristic Index explicitly?)

Reply:
I don't seem to understand, you have to sort based upon the output of the neural network, so that the best solver  is at index 0.

Reply:
To clarify It's correlated ID can be any index/value, but if you look at the loop in convolutionocl.cpp you'll see that we would want the best solver to be at index 0.

Reply:
Okay, because you've removed "pair" stuff and now again count on ordering of Solver IDs in the vector. Please comment out in the source code why we need sorting and let's assume this review comment is resolved, thanks!